docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,199 @@
+S3OutputModule
+==============
+
+### About Amazon S3
+-------------------
+
+**Amazon S3 (Simple Storage Service)** is an object storage system with a web service interface to store and retrieve any amount of data at any time from anywhere on the web, offered by Amazon Web Services.
+
+### Overview
+------------
+
+We can upload files into S3 through partitioning the AbstractFileOutputOperator. But, S3OutputModule is needed to parallel uploading the blocks of a file into S3. 
```

Comment:
Suggest rewriting:
File upload to S3 can also be done using `AbstractFileOutputOperator` but that operator uploads large files sequentially; the current module in contrast can substantially improve the upload speed of large files by reading and uploading its blocks in parallel. The table below lists additional benefints of this module over `AbstractFileOutputOperator`.